src/compiler/types.ts

```diff
@@ -1611,6 +1612,17 @@ namespace ts {
         getReferencedValueDeclaration(reference: Identifier): Declaration;
         getTypeReferenceSerializationKind(typeName: EntityName): TypeReferenceSerializationKind;
         isOptionalParameter(node: ParameterDeclaration): boolean;
+        getExportsOfModule(symbol: Symbol): Symbol[];
+        getDefiningTypeOfSymbol(symbol: Symbol): Type;
+        getTypeAtLocation(node: Node): Type;
+        resolveEntityName(entityName: EntityName | Expression, meaning: SymbolFlags, ignoreErrors?: boolean): Symbol;
+        getTypeWalker(accept?: (type: Type) => boolean): TypeWalker;
+    }
+
+    export interface TypeWalker {
+        visitType(type: Type): void;
+        visitTypeFromSymbol(symbol: Symbol): void;
+        reset(accept?: (type: Type) => boolean): void;

```

Comment:
Yeah.... as it turns out I never needed to reset the walker. But since it's on a public interface, I figured the extra utility would be useful for reusing the same walker, or resetting its visitor while walking. Do you want it gone?